scripts/end-to-end/verify-testcases.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+#
+# - SolidityEndToEndTest.trace was created with soltest with the following command on
+# develop @ commit 9318dae42cfd1a0f048665ec1eafae730bf7cb6b:
+#     ./soltest --log_level=test_suite -t SolidityEndToEndTest/ -- --no-smt
+#         --evmonepath /Users/alex/evmone/lib/libevmone.dylib --show-messages > SolidityEndToEndTest.trace
+# - a trace of the semantic tests can be created by using
+#     ./soltest --log_level=test_suite -t semanticTests/ -- --no-smt
+#         --evmonepath /Users/alex/evmone/lib/libevmone.dylib --show-messages > semanticTests.trace
```

Comment:
```suggestion
#     ./soltest --log_level=test_suite -t semanticTests/ -- --show-messages > semanticTests.trace
```
And wouldn't running on ``semanticTests/extracted/`` be enough?

Reply:
ah yes, thats true.. initially the directory structure was different.. I recently reintroduced `semanticTests/extracted/`